alti/lib/raster/georaster.py

```diff
@@ -98,9 +98,8 @@ def get_height_for_coordinate(self, x, y):
                 format_to_unpack = "<h"
             else:
                 format_to_unpack = "<l"
+        self.close_file()
         return unpack(format_to_unpack, self.file.read(self.data_size))[0]
```

Comment:
you might want to close the file after you read the height value.

```python
height = unpack(format_to_unpack, self.file.read(self.data_size))[0]
self.close_file()
return height
```

Reply:
yeah right... silly me... thanks for the feedback